src/compiler/scala/tools/nsc/settings/Warnings.scala

```diff
@@ -109,6 +109,7 @@ trait Warnings {
     val ImplicitNotFound       = LintWarning("implicit-not-found",        "Check @implicitNotFound and @implicitAmbiguous messages.")
     val Serial                 = LintWarning("serial",                    "@SerialVersionUID on traits and non-serializable classes.")
     val ValPattern             = LintWarning("valpattern",                "Enable pattern checks in val definitions.")
+    val EtaZero                = LintWarning("eta-zero",                  "Warn on eta-expansion (rather than auto-application) of zero-ary method.")
```

Comment:
one of us should add at the top to avoid collision at https://github.com/scala/scala/pull/7714/files#diff-6a32b09e0e5ebfd45d5af148012f231bR112.  Or maybe alphabetize.  Also have we said `Eta Candy` yet?

Reply:
Alphabetising + random names seems like a good collision avoidance mechanism, but then again, what are we optimising for?

Reply:
Probably we should optimi(sz)e for not dying.